Cargo.toml

```diff
@@ -25,7 +25,7 @@ lazy_static = "1.4.0"
 uuid = "1.1.2"
 
 flutter_rust_bridge_macros = { path = "frb_macros", version = "1" }
-flutter_rust_bridge = { path = "frb_rust", version = "1" }
+flutter_rust_bridge = { path = "frb_rust", version = "1", default-features = false }
```

Comment:
Hmm, so we are never testing the wasm-start feature in frb? Because IIRC we only test web for pure-dart, and now that one has wasm-start feature disabled.

Reply:
It's that way because flutter_rust_bridge is a workspace dependency, so specifying default-features on pure-dart has no effect. I could try to change pure-dart to depend by path, but not sure if that has any unintended effects.

Reply:
Totally agree. However, future PRs may make wasm-start more fancy etc, so it would be great to make it testable.

For example, is it possible to create one extra example as a sibling of pure_dart example, such that one example uses wasm-start (probably the default?) and another example (say example_without_wasm_start) forbids wasm start. Then, our pure_dart example captures the default case, while the example_without_wasm_start example tests when wasm-start disabled

Reply:
Another example crate might be overkill; we could use custom profiles in Cargo.toml to do this.